src/pages/PatientVitals.js

```diff
@@ -128,11 +130,18 @@ const PatientVitals = ({
           FOLLOWING_STATUS.pageNum === NEW_PATIENT_PAGES.covidHistory
         ? "page2-sub-wrapper"
         : "page3-sub-wrapper"
-      : FOLLOWING_STATUS.pageNum === 1
+      : messageType === MESSAGE_TYPES.dailyScreening
```

Comment:
As FOLLOWING_STATUS is using state page as it's value, it should be placed in useMemo:
const subWrapper = useMemo(() => ({ pageNum: page }), [paga]);

As subWrapper is using FOLLOWING_STATUS that depends on state it should be placed to useMemo too:
const subWrapper = useMemo(() => {
  if (messageType === MESSAGE_TYPES.newPatient) {
    if (FOLLOWING_STATUS.pageNum === NEW_PATIENT_PAGES.preExistingCondition ||
        FOLLOWING_STATUS.pageNum === NEW_PATIENT_PAGES.symptoms) {
       return "page1-sub-wrapper";
    } else if (...
  }, [FOLLOWING_STATUS]);

Reply:
resolved